tox.ini

```diff
@@ -9,46 +9,22 @@ commands =
 extras =
     testing
 
-[testenv:py2-cover]
-basepython = python2.7
-commands =
-    #coverage run setup.py -q test -q
-    pip install -e .[testing]
-    coverage run --source=hypatia {envbindir}/pytest
-    coverage xml -o coverage-py2.xml
-setenv =
-    COVERAGE_FILE=.coverage.py2
-extras =
-    testing
-
-[testenv:py3-cover]
-basepython = python3.10
+[testenv:coverage]
+basepython = python3.7
 commands =
     #coverage run setup.py -q test -q
     pip install -e .[testing]
     coverage run --source=hypatia {envbindir}/pytest
     coverage xml -o coverage-py3.xml
```

Comment:
```suggestion
    coverage xml -o coverage.xml
```

Reply:
Although I don't think that we currently use this file (and I don't believe that coverage uses this internally for coverage report, but I could be wrong). In the past this was was generated to provide coverage reporting in Jenkins.

Reply:
I will scrub the `coverage xml` bits later.